lib/services/total_price_for_products_in_basket.rb

```diff
@@ -0,0 +1,8 @@
+module Store
+  class TotalPriceForProductsInBasket
+    def call
+      basket = FetchBasket.new.call.map { |item| item[:total_price_with_vat] }
```

Comment:
FetchBasket will be better to replace for something more clear, like FetchItemsFromBasket